### DISCUSSION TITLE
release/v1.8.5分支中语音模型发布，不能在develop分支中使用了吗？

### DISCUSSION THREAD

**Discussion by DemoMoon** [PaddlePaddle/PaddleSpeech#582] [Q&A]
(venv) root@ip-10-0-1-86:baidu_en8k# bash server.sh /home/DeepSpeech/examples/baidu_en8k/ckpt/checkpoints/step-15
/home/DeepSpeech/examples/baidu_en8k/ckpt/checkpoints/step-15
Download language model ...
./data/lm/common_crawl_00.prune01111.trie.klm already exists, download skipped.
Download baidu_en8k_model model ...
data/pretrain/baidu_en8k_model_fluid.tar.gz already exists, download skipped.
mean_std.npz
README.md
vocab.txt
params.pdparams
/home/DeepSpeech/tools/venv/lib/python3.8/site-packages/paddle/fluid/layers/utils.py:26: DeprecationWarning: `np.int` is a deprecated alias for the builtin `int`. To silence this warning, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  def convert_to_list(value, n, name, dtype=np.int):
/home/DeepSpeech/tools/venv/lib/python3.8/distutils/__init__.py:4: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp
/home/DeepSpeech/tools/venv/lib/python3.8/site-packages/pkg_resources/_vendor/pyparsing.py:943: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.9 it will stop working
  collections.MutableMapping.register(ParseResults)
-----------  Configuration Arguments -----------
checkpoint_path: /home/DeepSpeech/examples/baidu_en8k/ckpt/checkpoints/step-15
config: conf/deepspeech2.yaml
device: cpu
dump_config: None
export_path: None
host_ip: localhost
host_port: 8080
nprocs: 1
opts: []
output: None
speech_save_dir: demo_cache
warmup_manifest: None
------------------------------------------------
data:
  augmentation_config: conf/augmentation.config
  batch_size: 64
  dev_manifest: data/manifest.dev
  keep_transcription_text: False
  max_duration: 27.0
  max_freq: None
  mean_std_filepath: data/mean_std.npz
  min_duration: 0.0
  n_fft: None
  num_workers: 0
  random_seed: 0
  shuffle_method: batch_shuffle
  sortagrad: True
  specgram_type: linear
  stride_ms: 10.0
  target_dB: -20
  target_sample_rate: 16000
  test_manifest: data/manifest.test
  train_manifest: data/manifest.train
  use_dB_normalization: True
  vocab_filepath: data/vocab.txt
  window_ms: 20.0
decoding:
  alpha: 1.9
  batch_size: 128
  beam_size: 300
  beta: 5.0
  cutoff_prob: 0.99
  cutoff_top_n: 40
  decoding_method: ctc_beam_search
  error_rate_type: cer
  lang_model_path: data/lm/zh_giga.no_cna_cmn.prune01244.klm
  num_proc_bsearch: 10
model:
  num_conv_layers: 2
  num_rnn_layers: 3
  rnn_layer_size: 1024
  share_rnn_weights: False
  use_gru: True
training:
  global_grad_clip: 5.0
  lr: 0.002
  lr_decay: 0.83
  n_epoch: 50
  weight_decay: 1e-06
-----------  Configuration Arguments -----------
checkpoint_path: /home/DeepSpeech/examples/baidu_en8k/ckpt/checkpoints/step-15
config: conf/deepspeech2.yaml
device: cpu
dump_config: None
export_path: None
host_ip: localhost
host_port: 8080
nprocs: 1
opts: []
output: None
speech_save_dir: demo_cache
warmup_manifest: data/manifest.test
------------------------------------------------
-----------------/home/DeepSpeech/examples/baidu_en8k/ckpt/checkpoints/step-15.pdparams
Traceback (most recent call last):
  File "/home/DeepSpeech/examples/baidu_en8k/../..//deepspeech/exps/deepspeech2/bin/deploy/server.py", line 134, in <module>
    main(config, args)
  File "/home/DeepSpeech/examples/baidu_en8k/../..//deepspeech/exps/deepspeech2/bin/deploy/server.py", line 100, in main
    start_server(config, args)
  File "/home/DeepSpeech/examples/baidu_en8k/../..//deepspeech/exps/deepspeech2/bin/deploy/server.py", line 55, in start_server
    model = DeepSpeech2Model.from_pretrained(dataset, config,
  File "/home/DeepSpeech/deepspeech/models/deepspeech2.py", line 238, in from_pretrained
    checkpoint.load_parameters(model, checkpoint_path=checkpoint_path)
  File "/home/DeepSpeech/deepspeech/utils/checkpoint.py", line 102, in load_parameters
    model_dict = paddle.load(params_path)
  File "/home/DeepSpeech/tools/venv/lib/python3.8/site-packages/paddle/framework/io.py", line 351, in load
    load_result = pickle.load(f) if six.PY2 else pickle.load(
_pickle.UnpicklingError: invalid load key, '\x00'.
Failed in starting demo server!

**Comment by DemoMoon** [comment]
就是直接拿BaiduEN8k Model的语音模型发布 在develop中推理用。

**Reply by zh794390558** [reply]
不可以这么用。1.8的只有lm还可以用。

**Comment by zh794390558** [comment]
已经重新发布了1.8的模型，适配到2.x上。 https://github.com/PaddlePaddle/DeepSpeech/tree/develop/examples/v18_to_v2x
